libs/subliminal_patch/providers/subssabbz.py

```diff
@@ -76,6 +78,14 @@ def get_matches(self, video):
              matches.add('hash')
 
         matches |= guess_matches(video, guessit(self.filename, {'type': self.type}))
+
+        if self.fps and video.fps and float(self.fps) != float(video.fps):
```

Comment:
Because framerate_equal() doesn't distinguish between 23.796, 23.798 and 24 fps. Over the course of 1.5 hour movie, this gives a 5-8 seconds difference. Check https://github.com/morpheus65535/bazarr/blob/development/libs/subliminal_patch/score.py#L26

Reply:
I understand but I'm not a fan of having a patch to some providers instead of correcting the source issue and having all providers benefit from it.

Reply:
There is nothing wrong with this function. I'm sorry you don't understand the reason why it's implemented this way.